### PR TITLE
YaruSection: simplify headline & remove headerWidget

### PR DESCRIPTION
**Pull request by jpnurmi**
It's easy enough to wrap stuff into a Row when needed, and then you'll have full control over the main & cross-axis alignment etc. ;-)

## Pull request checklist

- [x] this PR does not introduce visual changes

### CODE REVIEW

File: lib/src/pages/yaru_section.dart [hunk 1]
```diff
@@ -27,16 +27,13 @@ class YaruSection extends StatelessWidget {
...
   final double? height;
 
-  /// Aligns the widget horizontally along with headline.
-  ///
-  /// Both `headline` and `headerWidget` will be aligned horizontally
-  /// with [mainAxisAlignment] as [MainAxisAlignment.spaceBetween].
-  final Widget? headerWidget;
-
   /// The padding between the section border and its [child] which defaults to
-  /// `EdgeInsets.only(all: 8.0)`.
```
Comment (jpnurmi) [inline lib/src/pages/yaru_section.dart:37]: only all... oops :man_facepalming:

**Comment by Feichtmeier** [comment]
Since you kept the default text style it should be okay I think
Because this was the main pattern

--------------
| Headline |

| Children  |
----------------

**Comment by Feichtmeier** [comment]
Since you kept the default text style it should be okay I think
Because this was the main pattern

--------------
| Headline |

| Children  |
----------------

Edit: could you check if there is still the same header style?

**Comment by jpnurmi** [comment]
Yes, I kept the same default text style. There are no visual changes in the example:
image

**Comment by jpnurmi** [comment]
Or would you prefer to keep the original `headline` property name?

**Comment by Feichtmeier** [comment]
Ok I don't have a strong opinion 🐱 go with whatever you prefer then

**Comment by jpnurmi** [comment]
My English is not good enough to understand the real difference between "title" and "headline". I'm just used to one of them in Flutter... :rofl:

**Pull request merged**